subsys/shell/modules/kernel_service.c

```diff
@@ -107,13 +107,28 @@ static void shell_tdata_dump(const struct k_thread *cthread, void *user_data)
 		shell_print(shell, "\tTotal execution cycles: %" PRIu64 " (%u %%)",
 			    (uint64_t)rt_stats_thread.execution_cycles,
 			    pcnt);
+		shell_print(shell, "\tCurrent execution cycles: %" PRIu64,
+			    (uint64_t)rt_stats_thread.current_cycles);
+		shell_print(shell, "\tPeak execution cycles: %" PRIu64,
+			    (uint64_t)rt_stats_thread.peak_cycles);
+		shell_print(shell, "\tAverage execution cycles: %" PRIu64,
+			    (uint64_t)rt_stats_thread.average_cycles);
 #else
 		shell_print(shell, "\tTotal execution cycles: %u (%u %%)",
 			    (uint32_t)rt_stats_thread.execution_cycles,
 			    pcnt);
+		shell_print(shell, "\tCurrent execution cycles: %u",
```

Comment:
is there a reason why we need to distinguish that? Seems to me that this `#ifdef CONFIG_64BIT` could be removed as distinction between 64BIT and 32BIT got removed by https://github.com/zephyrproject-rtos/zephyr/commit/f169c5bc13707ad828a8ec34b3e9422f33e4861c

Reply:
Will look into that.